controllers/TuitController.ts

```diff
@@ -305,22 +300,17 @@ export default class TuitController implements TuitControllerI {
 
     /**
      * (For testing) Removes a tuit documents with tuit text that matches the
```

Comment:
lil thing, but "documents" should be singular

Reply:
This one has been reformulated a few times, and now the way it works is it deletes all tuits with the given tuit text, so I'll fix the javadoc line you pointed out so it makes sense and fix the function name accordingly